sycl/include/sycl/access/access.hpp

```diff
@@ -45,19 +45,12 @@ enum class placeholder { false_t = 0, true_t = 1 };
 enum class address_space : int {
   private_space = 0,
   global_space = 1,
-  constant_space = 2,
+  constant_space __SYCL2020_DEPRECATED("sycl::access::address_space::constant_"
+                                       "space is deprecated since SYCL 2020") =
+      2,
   local_space = 3,
   ext_intel_global_device_space = 4,
   ext_intel_global_host_space = 5,
```

Comment:
Is it in any way related to the host-device that was removed in SYCL2020 ? If so, should we put a deprecation here as well?

Reply:
Seems like it maps to {{opencl_global_host}} according to the [design document](https://github.com/intel/llvm/blob/sycl/sycl/doc/design/CompilerAndRuntimeDesign.md#address-spaces-handling), so I don't think it has any relation to the host device.